File: Python/Start/统计范围内的元音字符串数（Easy）.py
class Solution:
    def vowel_strings(self, words: List[str], left: int, right: int) -> int:
        vowels = set("aeiou")
        ans = 0
        for i in range(left, right + 1):
            if words[i][0] in vowels and words[i][-1] in vowels:
                ans += 1
        return ans

File: Cpp/Start/统计范围内的元音字符串数（Easy）.cpp
#include <vector>
#include <string>

class Solution 
{
public:
    bool is_vowel(char c)
    {
        if (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u') return true;
        return false;
    }

    int vowel_strings(std::vector<std::string>& words, int left, int right) 
    {
        int ans = 0;
        for (int i = left; i <= right; ++i)
        {
            int first_letter_is_vowel = is_vowel(words[i][0]); 
            int last_letter_is_vowel = is_vowel(words[i][words[i].size() - 1]);
            if (first_letter_is_vowel == 1 && last_letter_is_vowel == 1) ans++;
        }
        return ans;
    }
};